pyiso/caiso.py

```diff
@@ -283,7 +284,24 @@ def get_AS_dataframe(self, anc_region, latest=True, start_at=False, end_at=False
         MW columns holds $/MW float.
         If no data, returns an empty dataframe.
         """
-        # set args
+        # set args, handle latest differently than LMP b/c daily publication schedule
```

Comment:
That sounds perfect to me!
On Aug 28, 2015 12:34 PM, "ndavis6" <notifications@github.com> wrote:

> In pyiso/caiso.py
> <https://github.com/WattTime/pyiso/pull/38#discussion_r38235723>:
>
> > @@ -283,7 +284,24 @@ def get_AS_dataframe(self, anc_region, latest=True, start_at=False, end_at=False
> >          MW columns holds $/MW float.
> >          If no data, returns an empty dataframe.
> >          """
> > -        # set args
> > +        # set args, handle latest differently than LMP b/c daily publication schedule
>
> It could return a 1-hour (+ 1 second) window. That would guarantee some
> return and might be more consistent with the other functions.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WattTime/pyiso/pull/38/files#r38235723>.
>